### PR TITLE
DOC: antialiasing filter and link decimate resample, etc.

### PR DESCRIPTION
**Pull request by endolith**

### CODE REVIEW

File: scipy/signal/signaltools.py [hunk 1]
```diff
@@ -1786,8 +1786,8 @@ def resample(x, num, t=None, axis=0, window=None):
 
     See also
     --------
-    decimate
-    resample_poly
+    decimate : Downsample the signal after applying an anti-aliasing filter.
```
Comment (Eric89GXL) [inline scipy/signal/signaltools.py]: it's a bit weird to mention the anti-aliasing filter here but not for `resample` where it's also implicitly done by truncation or `resample_poly` where it's done explicitly

**Comment by Eric89GXL** [comment]
Otherwise LGTM

File: scipy/signal/signaltools.py (hunk 1 above)
Comment (endolith) [inline scipy/signal/signaltools.py]: True.  How else would you describe decimation vs the others?

Comment (Eric89GXL) [inline scipy/signal/signaltools.py]: As far as I can tell now `decimate` is just a special case of resampling, although it does allow for IIR antialiasing filters whereas `resample` and `resample_poly` do not. I'd just say `downsample a signal` or something and leave it at that

**Comment by endolith** [comment]
ok I changed the wording a bit

**Comment by Eric89GXL** [comment]
Looks good, I'll merge once the CIs come back happy

**Comment by Eric89GXL** [comment]
Thanks @endolith

**Pull request merged**